contribs/gmf/src/directives/partials/timeslider.html

```diff
@@ -0,0 +1,6 @@
+<div class="gmf-time-slider">
+  <div ui-slider="sliderCtrl.sliderOptions" ng-model="sliderCtrl.dates">
+    <span class="ui-slider-handle ui-state-default ui-corner-all" tabindex="0" style="left: 0%;"
```

Comment:
I dont' like to have `style` here. What is the comportment of the slider when the given default value isn't the first slider available value ?